utils/list_action.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'fetchers/fetcher'
+
+module Util
+  class ListAction < Trello::BasicData
+    extend Util::Fetcher
```

Comment:
Agreed. I will change that. Additionally there is already an `ActionFetcher` class. The reason I didn't quite use this yet is I wanted to write a generic one for all of the others to use. Currently the `ProgressChartsFetcher`, `StatsFetcher` and the list changes route need this same information. What I didn't like about the current `ActionFetcher` is that the callers always had to initially do the first call to `"/boards/:board_id"` and _then_ call into the `fetch_actions`. What I'd like to do is make this generic for all three cases and put my current specs around it instead.

That seem like a plan?